### ISSUE TITLE
Flex OrgvDC CPU and memory limit

### ISSUE THREAD

**Issue by nilic** [vmware/terraform-provider-vcd#488]
Hi,

I was testing the Flex OrgvDC type which was added in v2.7.0 of the provider.

Provider documentation for `vcd_org_vdc`, Compute Capacity section states the following:

> limit - (Optional) Capacity limit relative to the value specified for Allocation. It must not be less than that value. If it is greater than that value, it implies over provisioning. A value of 0 specifies unlimited units. Value in MB or MHz. Used with AllocationVApp ("Pay as you go") or **Flex (only for memory)**. 

and this indeed seems to be honored in the implementation - when memory limit is given as an input it appears in the plan, and when CPU limit is set, it seems to be ignored. Here is an attempt to change memory limit from 20GB to 0 and to set CPU limit to some arbitrary non-zero value for an existing OrgVDC:

```
      ~ compute_capacity {
            cpu {
                allocated = 10000
                limit     = 0
                overhead  = 0
                reserved  = 3000
                used      = 0
            }

          ~ memory {
                allocated = 20480
              ~ limit     = 20480 -> 0
                overhead  = 0
                reserved  = 6144
                used      = 0
            }
        }
```

So far so good. But, when I look at the created OrgVDC in the GUI, there is no mention of its memory limit, only CPU limit which is always "Unlimited" for OrgVDCs created through Terraform. Also, when creating a new OrgVDC, CPU limit (and not memory limit) is in the list of inputs:

![image](https://user-images.githubusercontent.com/6827325/78381134-353e9880-75d5-11ea-8215-f95aeb0bd6be.png)

Could this be a bug in the provider, where CPU limit should be applied to Flex OrgVDCs and memory limit ignored? Currently it's vice versa and that doesn't seem to match vCD logic. Thanks

**Comment by vbauzysvmware** [comment]
Hi @nilic, Thank you for information. I updating documentation - implementation is valid. There is no specific provider handling, API handles computing values.

**Comment by vbauzysvmware** [comment]
fixed and will be released in 3.1 version
